src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -1306,7 +1306,16 @@ rpmostree_sysroot_upgrader_flags_get_type (void)
           "allow-older" },
         { RPMOSTREE_SYSROOT_UPGRADER_FLAGS_DRY_RUN,
           "RPMOSTREE_SYSROOT_UPGRADER_FLAGS_DRY_RUN",
-          "dry-run" }
+          "dry-run" },
+        { RPMOSTREE_SYSROOT_UPGRADER_FLAGS_PKGCACHE_ONLY,
+          "RPMOSTREE_SYSROOT_UPGRADER_FLAGS_PKGCACHE_ONLY",
+          "pkgcache-only" },
+        { RPMOSTREE_SYSROOT_UPGRADER_FLAGS_SYNTHETIC_PULL,
+          "RPMOSTREE_SYSROOT_UPGRADER_FLAGS_SYNTHETIC_PULL",
+          "synthetic-pull" },
+        { RPMOSTREE_SYSROOT_UPGRADER_FLAGS_STAGE,
+          "RPMOSTREE_SYSROOT_UPGRADER_FLAGS_STAGE",
+          "stage" },
```

Comment:
Yeah, we should be able to use `glib-mkenums`, just requires some autogoo integration.  But short of that, mind adding a one-liner comment to the enum to note to update the gtype bits?  Feel free to r=me with that fixup.